testcontainers/src/core/image.rs

```diff
@@ -111,9 +111,9 @@ impl ContainerState {
         Self { ports }
     }
 
-    pub fn host_port(&self, internal_port: u16) -> u16 {
+    pub(crate) fn host_port_ipv4(&self, internal_port: u16) -> u16 {
```

Comment:
This has to stay public actually, otherwise the whole point of exposing it makes no sense :)

Sorry about the confusion!